Kitodo/src/main/java/org/kitodo/production/services/data/RulesetService.java

```diff
@@ -172,8 +162,8 @@ public Map<String, Object> findByFile(String file) throws DataException {
      * @return search result
      */
     List<Map<String, Object>> findByClientId(Integer clientId) throws DataException {
-        QueryBuilder query = createSimpleQuery(DocketTypeField.CLIENT_ID.getKey(), clientId, true);
-        return findDocuments(query);
+        // TODO delete method stub
+        throw new UnsupportedOperationException("no longer used function");
```

Comment:
Method is not marked as deprecated but is throwing this exception? Why not remove this method and maybe adjust the still used usage places?